### DISCUSSION TITLE
can I use express middlewares in nanoexpress for example helmet ?

### DISCUSSION THREAD

**Discussion by mostafaSalah12** [nanoexpress/nanoexpress#309] [Q&A]
I see nanoexpress has a very high performance, but I'm afraid to stuck in rewriting most of the existing middlewares, is it ok to use express middlewares in nanoexpress?

**Comment by dalisoft** [comment]
Yes, i think you need rewrite incompatible middlewares as nanoexpress does not cover all middlewares, only a few popular such as **cors** or **passport**
